packages/fxa-react/configs/tailwind.js

```diff
@@ -107,6 +107,12 @@ module.exports = {
       desktop: '1024px',
       desktopXl: '1441px',
     },
+    containers: {
```

Comment:
Since this is just JS I guess we could pull the object above into a constant and then spread it there and here to keep these two things consistent, but eh.